packages/core/src/Stores/client-store.js

```diff
@@ -2784,7 +2784,7 @@ export default class ClientStore extends BaseStore {
             return { key, subscription };
         } catch (error) {
             // eslint-disable-next-line no-console
-            console.warn(`Error: code = ${error?.error?.code}, message = ${error?.error?.message}`);
+            console.log(`Something wrong: code = ${error?.error?.code}, message = ${error?.error?.message}`);
```

Comment:
aren't we just hiding the error instead of fixing the underlying reason its stemming from? 
given the choice, I prefer the errors to be fully visible than hidden (unless this is genuinely irrelevant error)

Reply:
@wojciech-deriv The cause of error is we don't have `ZWL` currency in `exchange_rates` response from BE, but we are trying to subscribe to this currency in `p2p`. In my PR, I'm just handling it to prevent `TrackJS` error. We already discussed it many times, and fixing/showing the error to the user is out of the scope of this card. Firstly, we need to know, we really need to subscribe to the currency in p2p in buy/sell section? Secondly, why we are trying to subscribe so many times as we have rows in the table instead of subscribing one time? Let's fix TrackJS error first :)